nameserver/entry_test.go

```diff
@@ -17,67 +19,58 @@ func TestAdd(t *testing.T) {
 	entries := Entries{}
 	entries.add("A", "", router.UnknownPeerName, address.Address(0))
 	expected := Entries{
-		Entry{Hostname: "A", Origin: router.UnknownPeerName, Addr: address.Address(0)},
+		e(Entry{Hostname: "A", Origin: router.UnknownPeerName, Addr: address.Address(0)}),
 	}
 	require.Equal(t, entries, expected)
 
 	entries.tombstone(router.UnknownPeerName, func(e *Entry) bool { return e.Hostname == "A" })
 	expected = Entries{
-		Entry{Hostname: "A", Origin: router.UnknownPeerName, Addr: address.Address(0), Version: 1, Tombstone: 1234},
+		e(Entry{Hostname: "A", Origin: router.UnknownPeerName, Addr: address.Address(0), Version: 1, Tombstone: 1234}),
 	}
 	require.Equal(t, entries, expected)
 
 	entries.add("A", "", router.UnknownPeerName, address.Address(0))
 	expected = Entries{
-		Entry{Hostname: "A", Origin: router.UnknownPeerName, Addr: address.Address(0), Version: 2},
+		e(Entry{Hostname: "A", Origin: router.UnknownPeerName, Addr: address.Address(0), Version: 2}),
 	}
 	require.Equal(t, entries, expected)
 }
 
-func TestMerge(t *testing.T) {
-	e1 := Entries{
-		Entry{Hostname: "A"},
-		Entry{Hostname: "C"},
-		Entry{Hostname: "D"},
-		Entry{Hostname: "F"},
-	}
+func e(e Entry) Entry {
+	e.lHostname = strings.ToLower(e.Hostname)
+	return e
+}
```

Comment:
instead of this we could have `l(Entries) Entries`, which calls `Entries.addLowercase`. This could also be used in `makeEntries` below.